Tasks/SonarQubePostTest/Microsoft.VisualStudio.Services.CodeAnalysis.PostProcessor.dll.config

```diff
@@ -1,33 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is the old implementation of PRCA using a .net assembly which are moving away from.